src/scales/quantitativeScale.ts

```diff
@@ -152,7 +153,28 @@ export class QuantitativeScale<D> extends Scale<D, number> {
     });
     var newMin = minExistsInExceptions ? min : this.invert(this.scale(min) - (this.scale(max) - this.scale(min)) * p);
     var newMax = maxExistsInExceptions ? max : this.invert(this.scale(max) + (this.scale(max) - this.scale(min)) * p);
-    return this._niceDomain([newMin, newMax]);
+    if (this._niceDomainEnabled) {
+      return this._niceDomain([newMin, newMax]);
+    }
+    return ([newMin, newMax]);
+  }
+
+  /**
+   * Gets whether or not the scale will have a nice domain.
+   */
+  public niceDomain(): boolean;
```

Comment:
How about 
- nicifyDomain() (is this even a word?)
- prettifyDomain() (will it be confusing to use a synonym?)
- enableNiceDomain() (convention?) 

This was all I could think of. Do you like any of the suggestions?